test/testmouse.c

```diff
@@ -332,6 +332,9 @@ int main(int argc, char *argv[])
         return 0;
     }
 
+    /* Window won't show on Emscripten without this */
+    SDL_ShowWindow(window);
```

Comment:
Windows default to be shown, and there's an explicit ShowWindow call in SDL_FinishWindowCreation(), is that not working?

Reply:
This was due to a change of skipping `SDL_FinishWindowCreation` for Emscripten. Since the window wasn't being destroyed, I thought the call wasn't needed. I've reverted this. `SDL_ShowWindow` was always working as intended.